tests/Fixtures/code/BrokenClass.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php /** @noinspection SyntaxError */
```

Comment:
I would like to avoid as much as possible IDE-specific annotations, so please remove it